apps/src/music/views/MusicLabView.tsx

```diff
@@ -294,126 +285,19 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
     [blocklyWorkspace]
   );
 
-  // TODO: Rendering instructions with a function causes frequent re-renders.
-  // Pull this out into a component and/or drop support for "top" position
-  const renderInstructions = useCallback(
-    (position: InstructionsPosition) => {
-      const exemplarPlayerInsideInstructions =
-        AppConfig.getValue('exemplar-player-bottom') !== 'true';
-
-      const exemplarPlayer = exemplarSettings?.playerEnabled &&
-        exemplarSources &&
-        !isEditingExemplar && (
-          <ExemplarPlayerView
-            playbackEvents={exemplarPlaybackEvents}
-            title={exemplarSettings.playerTitle!}
-            player={player}
-            insideInstructions={exemplarPlayerInsideInstructions}
-          />
-        );
-
-      return (
-        <div
-          id="instructions-area"
-          className={classNames(
-            moduleStyles.instructionsArea,
-            position === InstructionsPosition.TOP
-              ? moduleStyles.instructionsTop
-              : moduleStyles.instructionsSide
-          )}
-        >
-          <PanelContainer
-            id="instructions-panel"
-            headerContent={musicI18n.panelHeaderInstructions()}
-            hideHeaders={hideHeaders}
-          >
-            <Instructions
```

Comment:
this will have a conflict with my pr 😄 https://github.com/code-dot-org/code-dot-org/pull/66845/files#diff-432ecbe88456facaecc77388cc0adf962b1ed1ba10c6a20794311156e6873799R335